gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -133,6 +142,7 @@ export const wrapCommand = (command) => {
       const proposalState = await this.fetchState(proposal)
       const isCreation = !proposalState
       if (isCreation) {
+        this.currentAction = ActionType.CREATE
```

Comment:
This is a bit hacky. `this.currentAction` will be a global variable that everyone has access to. I would either go with:
- `makeRawTransaction` accepts a `state` second parameter that comes determined from the execute function. Same as we did on Terra, but this require a bit more refactoring
- As we are logging the next action on each function, we could directly remove the [log](https://github.com/smartcontractkit/chainlink-solana/pull/270/files#diff-e137a7e73b6feadf1e43af5a461789ae132405b97460f422e156a88416ddd871R61), and prompt just `Continue?`